os/se/ameba/rtl_se_crypto_function.h

```diff
@@ -65,7 +65,7 @@
 #define SS_BASE_ADDRESS 0x14000
 #endif
 #ifdef CONFIG_AMEBASMART_TRUSTZONE
-#define SS_BASE_ADDRESS 0x1F0000 /* ToDo: Temporary set need changes based flash addr */
+#define SS_BASE_ADDRESS 0x1C000

```

Comment:
We should get this from CONFIG FLASH PART SIZE. Bcause we could move the location. (This is why we use the config.)

Reply:
This is ok to keep current.